.travis.yml

```diff
@@ -16,6 +16,26 @@ matrix:
   include:
     - python: "2.7"
       env:
+        - TOX_ENV=pythonbuild2.7
+        - TRAVIS_NODE_VERSION="0.12"
+    - python: "3.4"
+      env:
+        - TOX_ENV=pythonbuild3.4
+        - TRAVIS_NODE_VERSION="0.12"
+    - python: "3.5"
+      env:
+        - TOX_ENV=pythonbuild3.5
+        - TRAVIS_NODE_VERSION="0.12"
+    - python: "2.7"
```

Comment:
Maybe we should also be linting for 3.5 atleast. Not too familiar with discrepancies in the versions but `print` 'hello'` and `print('hello')` comes to mind.